const ArithmeticException = (message) => {
  this.message = message;
  this.name = "ArithmeticException";
}

export default {
  ArithmeticException
}